Traceback (most recent call last):
  File "main.py", line 11, in <module>
    prime_check([2, 44, 5, 3, 67, 17])
  File "main.py", line 6, in prime_check
    if a % i:
NameError: global name 'a' is not defined
